terraform/modules/rds-aurora-database/main.tf

```diff
@@ -0,0 +1,99 @@
+// setting the provider
+provider "aws" {
+  version = "~> 2.0"
+  region  = "ap-southeast-2"
+}
+
+
+// defining custom vpc
+// TODO: utilize the project's main VPC
+resource "aws_vpc" "temp_custom_vpc" {
+  cidr_block = "192.168.0.0/16"
+  tags = {
+    "Name" = "Terraform - temp_custom_vpc"
+  }
+}
+
+//defining custom subnet
+// TODO: utilize the project's subnetsh
+resource "aws_subnet" "temp_da_rds_subnet" {
+  vpc_id     = aws_vpc.temp_custom_vpc.id
+  cidr_block = "192.168.51.0/24"
+  availability_zone = "ap-southeast-2a"
+
+  tags = {
+    Name = "Terraform - temp_da_rds_subnet"
+  }
+}
+
+resource "aws_subnet" "temp_da_rds_subnet2" {
+  vpc_id     = aws_vpc.temp_custom_vpc.id
+  cidr_block = "192.168.52.0/24"
+  availability_zone = "ap-southeast-2b"
+
+  tags = {
+    Name = "Terraform - temp_da_rds_subnet2"
+  }
+}
```

Comment:
you can remove the "aws_subnet" resources. Let's use variables for the subnets on db_subnet_group.